REQUIREMENTS.rst

```diff
@@ -0,0 +1,72 @@
+============================
+ CESM software requirements
+============================
+
+Software requirements for installing, building and running CESM
+===============================================================
+
+Installing, building and running CESM requires:
+
+* a Unix-like operating system (Linux, AIX, OS X, etc.)
+
+* git client version 1.8 or newer
+
+* subversion client version ??? or newer
```

Comment:
@bandre-ucar documents 1.8.10 or later as being required for manage_externals. Do we truly need this recent of a svn client?

Reply:
What I documented is what I've regularly tested with. The oldest subversion I've tried is yellowstone, 1.6.11 and it works. manage_externals avoids scraping screen output for svn and relies on the return status and xml formatted status output. I don't know how far back those are stable.